src/main/java/Duke/Task/Deadline.java

```diff
@@ -0,0 +1,27 @@
+package main.java.Duke.Task;
+
+import java.time.LocalDate;
+
+public class Deadline extends Task {
+    public LocalDate deadline;
+
+    public Deadline(String task, boolean isComplete, LocalDate deadline) {
+        super(task, isComplete);
+        this.deadline = deadline;
+    }
+
+    /**
+     * Returns the description of the task and its completion in a string.
+     *
+     * @return Description of task in a String.
+     */
+    @SuppressWarnings("checkstyle:WhitespaceAfter")
```

Comment:
Instead of suppressing it, maybe you could try resolving this warning such that it does not pop up again. This statement would be redundant if this warning can be resolved manually.